tutor/routes/model.py

```diff
@@ -0,0 +1,14 @@
+from .. import app
+from ..controllers import model
+
+# example to get all models
+@app.route('/models')
+def models_route():
+    return model.getAllModels()
+
+
+# example of a route with different methods
+# example to return a json 
+@app.route('/model/store', methods=['GET', 'POST'])
+def store_book_route():
+    return model.createNewModel(modelName='model')
```

Comment:
not finding docs for this method either.

not sure why we seem to need to create a model object in order to return JSON.

Reply:
I didn't use comments because it just an example, and for that reason, I can return a boolean or every a string. The JSON is returned after the model was created, is to handle the created model - if we are willing to. 
I can return a JSON without creating a model, but I want to create a model to use the `/models` route.
All of that is an example, to check out the modeling system - will be removed after we will start using testing.

Reply:
if this is an example to help your team mates - If I can't understand it, how do you expect they will?

Reply:
The purpose is to write a document with more details, but I will change the comment to be more clear.